### PR TITLE
xfce.xfce4-time-out-plugin: init at 1.1.2

### PR DESCRIPTION
**Pull request by baronleonardo**
###### Description of changes
https://gitlab.xfce.org/panel-plugins/xfce4-time-out-plugin

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

@SuperSandro2000

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

File: pkgs/desktops/xfce/panel-plugins/xfce4-time-out-plugin/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  rev-prefix = "xfce4-time-out-plugin-";
+  odd-unstable = false;
+  sha256 = "sha256-xfkQjlUfvm0YXs3bRJD4W/71VkaPq3Y+cDFVNiL/bjc=";
+
+  buildInputs = [
+    gtk3 libxfce4ui libxfce4util xfce4-panel xfconf
+  ];
+
+  meta = with lib; {
+    description = "Xfce panel plugin acts like pomodoro";
```
Comment (romildo) [inline pkgs/desktops/xfce/panel-plugins/xfce4-time-out-plugin/default.nix:16]:
```suggestion
    description = "Panel plug-in to take periodical breaks from the computer";
```

**Review by romildo — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
> Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md?rgh-link-date=2022-10-03T18%3A48%3A35Z#submitting-changes) when naming your commits.

This must be fixed before we can think about merging this PR after romildo gave a green light.

**Comment by baronleonardo** [comment]
@SuperSandro2000 can't we just squash the commits into one when merging with a proper commit ?

**Review by romildo — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
We can do that but it is always nice if that is not required to do and it does not get forgotten. Also ofborg is not triggered automatically when the commit is not named correct.

**Comment by SuperSandro2000** [comment]
@ofborg build xfce.xfce4-time-out-plugin

**Pull request merged**